sml_small/editing/thousand_pounds/thousand_pounds.py

```diff
@@ -48,23 +69,45 @@ def run(
     error_ratio = None
     do_adjustment = False
     try:
-        predictive_value = determine_predictive_value(predictive, auxiliary)
+        input_parameters = validate_input(
+            predictive,
+            auxiliary,
+            principal_variable,
+            lower_limit,
+            upper_limit,
+            target_variables,
+        )
+        predictive_value = determine_predictive_value(
+            input_parameters[InputParameters.PREDICTIVE.value],
+            input_parameters[InputParameters.AUXILIARY.value],
+        )
         if (
             predictive_value
         ):  # Allow the case where given predictive = 0 and aux is missing to be a valid case
```

Comment:
Does this comment make sense here (I know it wasn't yours), especially now we have a validation function ?

Reply:
Amended, Comment removed